src/uwtools/tests/api/test_fv3.py

```diff
@@ -25,30 +23,12 @@ def test_execute(tmp_path):
 
 
 def test_graph():
-    @external
-    def ready():
-        yield "ready"
-        yield asset("ready", lambda: True)
-
-    ready()
-    assert fv3.graph().startswith("digraph")
+    with patch.object(fv3.support, "graph") as graph:
+        fv3.graph()
+    graph.assert_called_once_with()
```

Comment:
This is just a plumbing test now -- the real test is in `uwtools.tests.drivers.test_support`.